Makefile

```diff
@@ -211,11 +211,16 @@ generate-bundle: operator-sdk manifests kustomize ## Generate bundle manifests a
 	$(OPERATOR_SDK) bundle validate $(BUNDLE_DIR)
 	./hack/ignore-createdAt-bundle.sh
 
+.PHONY: bundle-openshift
+bundle-openshift:
```

Comment:
I will ask again :) Why do we need to have two targets? 

I would advocate for a single target and make sure the two bundles are always generated and up-to-date simultaneously (also the makefile will be simpler).

Reply:
Done! 1 target to rule them all!

Reply:
:P